dev_environment/README.md

```diff
@@ -42,14 +42,18 @@ Here is a visual overview:
   `hey -n 200 -c 20 http://localhost:18080`. This command should return quickly
   and result in successful HTTP responses (200 OK).
 * Run `vagrant ssh`, this will log you into the virtual machine
-* On the VM, go to `~/code/l3afd` and run `go install .`
+* On the VM, go to `~/code/l3afd` and run `make install`
 * On the VM, go to `~/go/bin` and run `l3afd` as root:
   `sudo ./l3afd --config /vagrant/cfg/l3afd.cfg`
 * On the host, configure L3AFD to execute sample eBPF programs by running
   `curl -X POST http://localhost:37080/l3af/configs/v1/update -d
   "@cfg/payload.json"`.  The `payload.json` file can be inspected and modified
   as desired. For more information on the L3AFD API see the [L3AFD API
   documentation](https://github.com/l3af-project/l3afd/tree/main/docs/api).
+* L3AFD can also be configured to execute sample eBPF programs from
+  Swagger UI `http://localhost:37080/swagger/index.html` on the host machine.
```

Comment:
What permissions are necessary to execute sample eBPF programs this way?
Can an unprivileged local user use the swagger UI to gain root privileges?  If so, that would be an elevation of privilege vulnerability.

Reply:
This is exactly the same thing as we are doing via curl request from the host machine to start the sample programs on the vagrant VM, but from the browser. And since this is a set up for DEV environment, TLS is disabled by default and there is no need of any certs also. So there is no change/addendum to any permissions.